sdk/core/azure-core/CHANGELOG.md

```diff
@@ -8,6 +8,8 @@
 
 ### Bugs Fixed
 
+- Updated field type `CurlTransportOptions.Proxy` from `std::string` to `Azure::Nullable<std::string>`. This change allow to se an empty string to make libcurl ignore proxy settings from environment [](https://github.com/Azure/azure-sdk-for-cpp/issues/3537).
```

Comment:
There's a typo here.

se -> see

Will fix as part of https://github.com/Azure/azure-sdk-for-cpp/pull/3598